session/server.go

```diff
@@ -1,13 +1,15 @@
 package session
 
 import (
+	"context"
 	"crypto/tls"
 	"fmt"
 	"sync"
 	"time"
 
 	"github.com/btcsuite/btcd/btcec/v2"
 	"github.com/lightninglabs/lightning-node-connect/mailbox"
+	"github.com/lightninglabs/taproot-assets/fn"
```

Comment:
I'm not sure if we want to use this package or the lnd one?

Reply:
yeah can switch the the lnd one 👍